File: CompilationTest/FakeEnumTest/results/FakeEnumTest.ts
class TokenType {
    static endToken: string = "EndToken";
    static whitespace: string = "Whitespace";
    static identifier: string = "Identifier";
    static operatorX: string = "Operator";
    static noInitializer: string;
}

const casingTest = TokenType.endToken;
console.log(casingTest);

File: CompilationTest/FakeEnumTest/results/FakeEnumTest.java
class TokenType {
    public static String endToken = "EndToken";
    public static String whitespace = "Whitespace";
    public static String identifier = "Identifier";
    public static String operatorX = "Operator";
    public static String noInitializer;
}

class Program {
    public static void main(String[] args) throws Exception {
        String casingTest = TokenType.endToken;
        System.out.println(casingTest);
    }
}